build-info-extractor/src/main/java/org/jfrog/build/extractor/clientConfiguration/client/artifactory/services/DeleteBuilds.java

```diff
@@ -0,0 +1,50 @@
+package org.jfrog.build.extractor.clientConfiguration.client.artifactory.services;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpRequestBase;
+import org.jfrog.build.api.util.Log;
+import org.jfrog.build.extractor.clientConfiguration.client.VoidJFrogService;
+
+import java.io.IOException;
+
+/**
+ * Delete multiple build numbers of a certain build.
+ */
+public class DeleteBuilds extends VoidJFrogService {
+    public static final String DELETE_BUILDS_ENDPOINT = "api/build/";
+    private final String buildName;
+    private final boolean deleteArtifact;
```

Comment:
deleteArtifact --> deleteArtifacts
(Let's also change the local variables' names).